dockerfiles/centos/centos-7/centos-7-builder/README

```diff
@@ -0,0 +1,2 @@
+The Dockerfile for this directory is autogenerated based on dockerfiles/templates/Dockerfile.builder
+Do not place an actual Dockerfile here.
```

Comment:
We should add the -builder directories to the gitignore, that will discourage people from putting Dockerfiles there and help avoid the generated files being added to the repo.